components/suggestion/command_provider/command_provider.tsx

```diff
@@ -74,9 +76,11 @@ export default class CommandProvider extends Provider {
             return false;
         }
         if (UserAgent.isMobile()) {
-            return this.handleMobile(pretext, resultCallback);
+            this.handleMobile(pretext, resultCallback);
```

Comment:
I understand this was here from before. Just wondering what is the difference between mobile and not mobile. This is just the browser view on mobile, not the native app, so not sure why there is a big differentiation.

Reply:
There is a different endpoint being used for mobile web, which just needs to change to use the correct endpoint afaik. Here's a conversation on this https://community-daily.mattermost.com/core/pl/7fmqsg8bpfyi8ccentse9s3nry

Reply:
I've made App commands supported for both platforms now, and have asked why these were split into desktop and mobile here https://community-daily.mattermost.com/core/pl/dfsxuqkrypd18mg8ydfdqkcp6w